src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -318,6 +318,10 @@
     text: |
         az monitor activity-log alert update --enabled true --name MyActivityLogAlerts --resource-group MyResourceGroup --subscription MySubscription
     crafted: true
+  - name: Update the details of this activity log alert. (autogenerated)
+    text: |
+        az monitor activity-log alert update --name MyActivityLogAlerts --resource-group MyResourceGroup --tags key[=value]
```

Comment:
Should we provide specific keys and values for --tags parameter in example?

Reply:
Yeah, we should. Could you add suggestions and we could quickly apply changes in Github

Reply:
```suggestion
        az monitor activity-log alert update --name MyActivityLogAlerts --resource-group MyResourceGroup --tags key=value
```